anesthetic/plot.py

```diff
@@ -183,6 +183,7 @@ def make_2d_axes(params, **kwargs):
                     axes[x][y].twin = axes[x][y].twinx()
                     axes[x][y].twin.set_yticks([])
                     axes[x][y].twin.set_ylim(0, 1.1)
+                    axes[x][y].set_zorder(axes[x][y].twin.get_zorder() + 1)
```

Comment:
Why is this necessary?

Reply:
> * Set axes in front of axes.twin.
>   - Since legends are drawn on axes, they got hidden behind the lines and histograms which belonged to the twin axes and thus were in front.

The twin axes get drawn after the original axes, but the legends are drawn on the original axes and will be hidden behind the twin axes. Actually anything added to those axes will dissappear behind the the twin axes, e.g. if someone were to try to annotate something they'd have a hard time without knowing about the twin.

The change just puts the original in front of the twin.

Reply:
Sounds reasonable